packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -89,4 +89,14 @@ export const keys = {
   ENTER: 'Enter',
   ARROWUP: 'ArrowUp',
   ARROWDOWN: 'ArrowDown',
-};
\ No newline at end of file
+};
+
+export function getItemfromLocalStorage(key) {
+  const value = localStorage.getItem(key);
+  return value !== 'undefined' ? JSON.parse(value) : null;
+}
+
+export function setIteminLocalStorage(key, item) {
```

Comment:
I'd suggest to rename `setIteminLocalStorage` to `setItemInLocalStorage`. It is a little easier to read.